scripts/copy_output.lua

```diff
@@ -53,6 +54,11 @@ for _, image in ipairs(lib.images) do
 	end
 end
 
+local kernel_debug_source = string.format('openwrt/bin/targets/%s/kernel-debug.tar.bz2',
+	bindir)
+local kernel_debug_dest   = env.GLUON_IMAGEDIR..'/debug/'..target..'-kernel-debug.tar.bz2'
```

Comment:
Lining up the `=` characters doesn't make much sense when the line before is wrapped.

Reply:
You are totally right. I have removed the alignment.